lisa/sut_orchestrator/azure/features.py

```diff
@@ -969,12 +971,29 @@ def _select_vm_size(
 
 
 class Hibernation(AzureFeatureMixin, features.Hibernation):
-    @classmethod
-    def on_before_deployment(cls, *args: Any, **kwargs: Any) -> None:
-        # arm_parameters: AzureArmParameter = kwargs.pop("arm_parameters")
-        # arm_parameters.enable_hibernation = True
-        ...
+    _hibernation_properties = """
+        {
+            "additionalCapabilities": {
+                "hibernationEnabled": "true"
+            }
+        }
+        """
 
     def _initialize(self, *args: Any, **kwargs: Any) -> None:
         super()._initialize(*args, **kwargs)
         self._initialize_information(self._node)
+
+    @classmethod
+    def _enable_hibernation(cls, *args: Any, **kwargs: Any) -> None:
+        parameters: Any = kwargs.get("arm_parameters")
+        if parameters.use_availability_sets:
```

Comment:
When will it be created? If the availability is used by default, how to test hibernation?

Reply:
When specify the availability set in runbook, then use_availability_sets is true, by default use_availability_sets is set as false. 
should we remove the availability set, if it is specified in runbook.
Or we can have a pipeline to run hibernation to make sure the coverage.